### DISCUSSION TITLE
problems with yi hack integration in home assistant

### DISCUSSION THREAD

**Discussion by divertipo** [roleoroleo/yi-hack_ha_integration#157] [General]
Thanks for the tutorial and clear information. I've cahanged firmwate to the yi home 1080 camera and I can access via web. BUT when I try to install the script in HA configuration running this script: 
$ git clone https://github.com/roleoroleo/yi-hack_ha_integration.git
system seems to work but it doesnt appear in custom components the files listed: 
![Captura](https://github.com/user-attachments/assets/b6f6a409-6778-4002-9fea-938f75fd834c)

So, I'm not able to make integration in HA
Maybe is a stupid mistake but I'm completly stuck. Any help, please?

**Comment by roleoroleo** [comment]
Try to follow the readme: https://github.com/roleoroleo/yi-hack_ha_integration
You should download the last release from here https://github.com/roleoroleo/yi-hack_ha_integration/releases/tag/0.5.2
and decompress it in the configuration directory of your home assistant.
Restarting ha you should find it in the integration list.
